XenCenterLib/Archive/ZipArchiveWriter.cs

```diff
@@ -38,17 +38,17 @@ namespace XenCenterLib.Archive
 {
     class DotNetZipZipWriter : ArchiveWriter
     {
-        private ZipOutputStream zip = null;
+        private ZipOutputStream zip;
         private bool disposed;
 
-        public DotNetZipZipWriter(Stream outputStream) : this()
+        public DotNetZipZipWriter(Stream outputStream)
         {
             zip = new ZipOutputStream( outputStream ) {EnableZip64 = Zip64Option.AsNecessary};
+            disposed = false;

```

Comment:
This is not really needed, but I left it there for symmetry with what the constructors in the other archive classes do.